File: Data Structures & Algorithms/grumpy-bookstore-owner/submission-0.py
class Solution:
    def maxSatisfied(self, customers: List[int], grumpy: List[int], minutes: int) -> int:
        res, n = 0, len(customers)

        for i in range(n):
            if not grumpy[i]:
                res += customers[i]
        
        satisfied = res
        for i in range(n - minutes + 1):
            cur = 0
            for j in range(i, i + minutes):
                if grumpy[j]:
                    cur += customers[j]
            res = max(res, satisfied + cur)
        
        return res

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int res = 0;
        int temp = 0;
        for (int i : nums){
            if (i == 1){
                temp++;
            } else{
                temp = 0;
                res = Math.max(temp, res);
            }

            res = Math.max(temp, res);
        }
        return res;
    }
}   